tests/test_charts.py

```diff
@@ -467,35 +467,39 @@ def test_multicomp_plot_common(plot_f, request):
         (1.0, 0.4980392156862745, 0.0, 1.0),
         (0.6509803921568628, 0.33725490196078434, 0.1568627450980392, 1.0),
     ]
-    colors = [(1, 0, 1, 1), (0, 1, 1, 1), (1, 1, 0, 1)]
+    colors = [(1.0, 0.0, 1.0, 1.0), (0.0, 1.0, 1.0, 1.0), (1.0, 1.0, 0.0, 1.0)]
     labels = ["Foo", "Spam", "Bla"]
 
     plot.color_scheme = cs
     assert plot.color_scheme == cs
     assert plot._color_series.GetColorScheme() == plot.COLOR_SCHEMES[cs]["id"]
-    assert np.allclose(plot.colors, cs_colors)
-    series_colors = [plot._from_c3ub(plot._color_series.GetColor(i)) for i in range(len(cs_colors))]
+    assert all(pc == cs for pc, cs in zip(plot.colors, cs_colors))
+    series_colors = [
+        pyvista.Color(plot._color_series.GetColor(i)).f_rgba for i in range(len(cs_colors))
+    ]
     assert np.allclose(series_colors, cs_colors)
     lookup_colors = [plot._lookup_table.GetTableValue(i) for i in range(len(cs_colors))]
     assert np.allclose(lookup_colors, cs_colors)
-    assert np.allclose(plot.brush.color, cs_colors[0])
+    assert plot.brush.color == cs_colors[0]
 
     plot.colors = None
     assert plot.color_scheme == plot.DEFAULT_COLOR_SCHEME
     plot.colors = cs
     assert plot.color_scheme == cs
     plot.colors = colors
-    assert np.allclose(plot.colors, colors)
-    series_colors = [plot._from_c3ub(plot._color_series.GetColor(i)) for i in range(len(colors))]
+    assert all(pc == c for pc, c in zip(plot.colors, colors))
+    series_colors = [
+        pyvista.Color(plot._color_series.GetColor(i)).f_rgba for i in range(len(colors))
+    ]
     assert np.allclose(series_colors, colors)
     lookup_colors = [plot._lookup_table.GetTableValue(i) for i in range(len(colors))]
     assert np.allclose(lookup_colors, colors)
-    assert np.allclose(plot.brush.color, colors[0])
+    assert plot.brush.color == colors[0]
 
     plot.color = colors[1]
-    assert np.allclose(plot.color, colors[1])
-    assert np.allclose(plot.colors, [colors[1]])
-    assert np.allclose(plot.brush.color, colors[1])
+    assert np.allclose(plot.color.f_rgba, colors[1])
+    assert len(plot.colors) == 1 and np.allclose(plot.colors[0].f_rgba, colors[1])
```

Comment:
It's basically a check to make sure the plot's `color` and `colors` properties "stay in sync". More precisely, we want `plot.color == plot.colors[0] == colors[1]` (and `len(plot.colors) == 1`). And now that I have written this down, I realize that is the better check to do, instead of resorting to `f_rgba` comparisons :p